modules/transformer-kinesis/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/transformer/kinesis/parquet/ParquetSchema.scala

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2012-2021 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and
+ * limitations there under.
+ */
+package com.snowplowanalytics.snowplow.rdbloader.transformer.kinesis.parquet
+
+import com.github.mjakubowski84.parquet4s.SchemaDef
+import com.snowplowanalytics.iglu.schemaddl.parquet.{Field, Type}
+import com.snowplowanalytics.snowplow.rdbloader.common.transformation.parquet.fields.AllFields
+import org.apache.parquet.schema.LogicalTypeAnnotation._
+import org.apache.parquet.schema.PrimitiveType.PrimitiveTypeName._
+import org.apache.parquet.schema.{LogicalTypeAnnotation, MessageType, Types, Type => PType}
+
+private [parquet] object ParquetSchema {
+
+  def build(allFields: AllFields): MessageType = {
+    val fieldsOnly = allFields.fieldsOnly
+    val types = fieldsOnly.map(asParquetField)
+   Types.buildMessage()
+      .addFields(types: _*)
+      .named("snowplow")
+  }
+
+  private def asParquetField(field: Field): PType = {
+    val normalizedName = Field.normalize(field).name
+
+    asParquetType(field.fieldType)
+      .withRequired(field.nullability.required)
+      .apply(normalizedName)
+  }
+
+  private def asParquetType(fieldType: Type): SchemaDef = fieldType match {
+    case Type.String =>
+      SchemaDef.primitive(BINARY, logicalTypeAnnotation = Some(stringType()))
+    case Type.Json => 
+      SchemaDef.primitive(BINARY, logicalTypeAnnotation = Some(stringType())) //TODO we could use json logical type
```

Comment:
I saw you added the json logical type and then removed it again.  I guess you removed it to be consistent with Spark?

I think we should use the json logical type!  I like it.  And I checked already that Spark just reads it as a string so I think it should get loaded into Databricks OK.